examples/3d/lighting.rs

```diff
@@ -1,25 +1,12 @@
 use bevy::{
-    core::Time,
     diagnostic::{FrameTimeDiagnosticsPlugin, LogDiagnosticsPlugin},
     ecs::prelude::*,
-    input::Input,
-    math::{Quat, Vec3},
-    pbr2::{
-        AmbientLight, DirectionalLight, DirectionalLightBundle, PbrBundle, PointLight,
-        PointLightBundle, StandardMaterial,
-    },
-    prelude::{App, Assets, BuildChildren, KeyCode, Transform},
-    render2::{
-        camera::{OrthographicProjection, PerspectiveCameraBundle},
-        color::Color,
-        mesh::{shape, Mesh},
-    },
-    PipelinedDefaultPlugins,
+    prelude::*,
 };
 
 fn main() {
     App::new()
-        .add_plugins(PipelinedDefaultPlugins)
+        .add_plugins(DefaultPlugins)
         .add_plugin(FrameTimeDiagnosticsPlugin::default())
```

Comment:
Should the examples still have the diagnostics plugins by default?

Reply:
Nope good call. Its just unnecessary cruft.